### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by HarryCusworth**
actual text isnt written, this is just place holder.

colours could still use work but basic format is down.

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -1 +1,104 @@
...
+	<link rel="stylesheet" type="text/css" href="normalize.css">	
+</head>
 
+<body>
+	<div class="splitGrid">
+		
+		<!--main body of the page content -->
+		<div class="bodyExp">	
+				<section class="careerProfile">
+					<div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> Career Profile</div>
```
Comment (norayna) [inline portfolio.html]: Put line breaks in between the tags so it's easier to read

File: portfolio.html [hunk 2]
```diff
@@ -1 +1,104 @@
...
+					<div class="profileBody"><p> I decided tyo learn to code after years of frustrating experience with charity funding, instead of using all my energy to secure funding for projects that do good I decided to learn to code so that I could actually design and deliver digital projects that do good and that can be self sufficient. Designing solutions that don't require dependence on increasingly scarce resources.
```
Comment (norayna) [inline portfolio.html]: Line break after the div opening tag, in fact anywhere where you have multiple nested tags you should put a line break in between

File: portfolio.html [hunk 3]
```diff
@@ -1 +1,104 @@
...
+					</p>
+					</div>
+				</section>
+		
+				<section class="experience"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> Experience</div>
+						
+							<div class=expSpan>
```
Comment (norayna) [inline portfolio.html]: the class name `expSpan` is a bit confusing - the first half probably means experience and the second half I don't know, because this is a `div` and not a `span`. Wherever you write a class name that is related to "experience", you should use the full word instead of shortening it to avoid ambiguity and confusion about the meaning of "exp".

File: portfolio.html [hunk 4]
```diff
@@ -1 +1,104 @@
...
+					<div class="contactUnit"><i class="fab fa-linkedin fa-1.5x" style="color:black"></i><a href="https://www.linkedin.com/in/harry-cusworth-83b56a182/"> Linkedin</a></div>
+					<div class="contactUnit"><i class="fab fa-github fa-1.5x" style="color:black"></i> <a href="https://github.com/HarryCusworth">GitHub</a></div>
+				
+			</section>
+			
+	
+			<section class="educationDeets">
+					<h3>Education Details</h3>
+					
+						<!-- <div class="edUnit"> -->
```
Comment (norayna) [inline portfolio.html]: All commented out code should be removed after you've determined that it's not needed. For best practice, we try not to allow any commented out code to make it into the repo at all. If we ever need to bring old code back, we should be able to find it in old commits.

tldr - remove all commented out code please

File: portfoliostylesheet.css [hunk 5]
```diff
@@ -0,0 +1,332 @@
+* {
+	box-sizing:border-box;
```
Comment (norayna) [inline portfoliostylesheet.css]: missing space

File: portfoliostylesheet.css [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+    color: #B4535A;
+}
+
+a:hover {
+    color: #D0D2CA;
+}
+
+html { 
+	font-size:16px;
+	font-family:"verdana", sans-serif;
```
Comment (norayna) [inline portfoliostylesheet.css]: missing space again (check everywhere)

File: portfoliostylesheet.css [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+body {
+	margin:0px;
+	padding:0px;
+}
+	
+h3 {
+	font-family:"avenir book";
+	text-size:1rem;
```
Comment (norayna) [inline portfoliostylesheet.css]: Using `rem` for font size but you haven't set the base font size in pixels - so it will depend on the font size set in `normalize.css`. That means it will probably be consistent between browsers but you might want to set it yourself just to make it clear what the base font size is.

File: portfoliostylesheet.css [hunk 8]
```diff
@@ -0,0 +1,332 @@
...
+	color:#B28537;
+}
+
+.splitGrid {
```
Comment (norayna) [inline portfoliostylesheet.css:33]: This class name describes exactly what the thing looks like, rather than the purpose that it serves on the page. A better name would be something that describes its purpose on the page.

I would even suggest just using a `main` tag because that's what this is on the page.

File: portfoliostylesheet.css [hunk 9]
```diff
@@ -0,0 +1,332 @@
...
+	display: grid;
+	grid-template-columns: 4fr 1fr ;
```
Comment (norayna) [inline portfoliostylesheet.css:35]: rogue space

File: portfoliostylesheet.css [hunk 10]
```diff
@@ -0,0 +1,332 @@
...
+	margin-left: 15%;
+	margin-right:15%;
+}
+
+
+.footer {
+	margin-left:40%;
```
Comment (norayna) [inline portfoliostylesheet.css]: Use `margin: 0 auto` to center it instead

File: portfoliostylesheet.css [hunk 11]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+.langSkills {
+	display:flex;
+	flex-direction:column;
+	align-items:left;
+/*	padding-left:25px;*/
+	padding-right:25px;
+	margin-left:15%;
+	margin-right:5px;
```
Comment (norayna) [inline portfoliostylesheet.css]: This is some highly specific maths using both margins and padding and different units of measurement - do you think there might be a simpler way of achieving the same thing?

Also, a lot of the code for this class is also used for some other classes. That means that if you change it here, you have to go and change it in other places too. Instead, put all the common css together like so:
```css
.contactDeets, .educationDeets, .langSkills {
   /* common code */
}

Comment (norayna) [reply portfoliostylesheet.css]: And for the code that's different just use separate selectors like before

File: portfoliostylesheet.css [hunk 12]
```diff
@@ -0,0 +1,332 @@
...
+	padding-bottom:100px;
+	padding-top:20px;
+}
+
+.contactUnit{
+	color:#B4535A;
+	font-size:.9rem;
+	focus:;
```
Comment (norayna) [inline portfoliostylesheet.css]: empty property

File: portfoliostylesheet.css [hunk 13]
```diff
@@ -0,0 +1,332 @@
...
+		flex-wrap:wrap;
+		align-items:center;
+		justify-content:flex-start;
+	}
+	
+	.footer {
+		margin:0;
+	}
+	
+	}
```
Comment (norayna) [inline portfoliostylesheet.css:340]: incorrect indentation

**Review by norayna — CHANGES_REQUESTED** [review]
General comments:
- The name of the PR should describe what you've done in it (a bit like a commit message but for the whole PR), e.g. "Created and styled the whole page" or something else
- the mobile view seems to be broken - this is exactly how it looks in my browser (Firefox). There is a horizontal scrollbar and the rest of the blue rectangle is further to the right
image

Is it supposed to be ready for this review?

File: portfoliostylesheet.css (hunk 7 above)
Comment (HarryCusworth) [reply portfoliostylesheet.css]: ummm i did set a fony size in the html tag of the css stylesheet line 17

File: portfoliostylesheet.css (hunk 10 above)
Comment (HarryCusworth) [reply portfoliostylesheet.css]: this didn't work, I think we talked about it the other day as being a confusing thing.
